### PR TITLE
Include exercism local tooling webserver in Docker image

### PR DESCRIPTION
**Pull request by ErikSchierboom**
CC @joshgoebel

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,4 +19,6 @@ RUN dotnet publish -r linux-musl-x64 -c Release -o /opt/test-runner
 FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine AS runtime
 WORKDIR /opt/test-runner
 COPY --from=build /opt/test-runner/ .
+COPY --from=build /usr/local/bin/ /usr/local/bin/
```
Comment (joshgoebel) [inline Dockerfile:22]: Probably a little wide, but I'm guessing so far our little web server is the only thing in /user/local/bin? :-)

Comment (ErikSchierboom) [reply Dockerfile:22]: Yep :)

Comment (joshgoebel) [reply Dockerfile:22]: Wait does this need the `EXEC true` hack in-between? @iHiD

Comment (iHiD) [reply Dockerfile:22]: Apparently not (as it built) 🤷

**Pull request merged**